Gulpfile.js

```diff
@@ -199,22 +199,28 @@ var lintScripts = function(done) {
   done()
 }
 
-// pretty print all xml listings
-// articles not yet decided
-var prettyXml = function(done) {
-  src(paths.xml.listings, { base: "./" })
-  .pipe(muxml({
-    stripComments: false,
-    stripCdata: false,
-    stripInstruction: false,
-    saxOptions: {
-      trim: true,
-      normalize: true
-    }
-  }))
-    .pipe(dest("./"))
-    // Signal completion
-    done()
+function prettyPrintXml(xmlSrcPath, xmlTargetPath, done) {
+  src(xmlSrcPath, { base: "./" })
+      .pipe(muxml({
+        stripComments: false,
+        stripCdata: false,
+        stripInstruction: false,
+        saxOptions: {
+          trim: true,
+          normalize: true
+        }
+      }))
+      .pipe(dest(xmlTargetPath))
+  // Signal completion
+  done()
+}
+
+var prettyPrintListingsXml = function(done) {
```

Comment:
should be `const`

Reply:
I would be in favour of named function even

Reply:
Nothing else is const - so why that one?

Reply:
Well, that can be applied afterwards. It was a suggestion to apply a best-practice.